### PR TITLE
feat: filter support plain link

### PR DESCRIPTION
**Pull request by CorrectRoadH**
close: #1653

### CODE REVIEW

File: web/src/helpers/filter.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { PLAIN_LINK_REG } from "@/labs/marked/parser/PlainLink";
 import { getUnixTimeMillis } from "./datetime";
 import { TAG_REG, LINK_REG } from "@/labs/marked/parser";
```
Comment (Zeng1998) [inline web/src/helpers/filter.ts:4]:
```suggestion
import { TAG_REG, LINK_REG, PLAIN_LINK_REG } from "@/labs/marked/parser";
```

https://github.com/usememos/memos/blob/main/web/src/labs/marked/parser/index.ts#L22

or

```suggestion
import { PLAIN_LINK_REG } from "@/labs/marked/parser/PlainLink";
import { LINK_REG} from "@/labs/marked/parser/Link";
import { TAG_REG } from "@/labs/marked/parser/Tag";
```

**Review by Zeng1998 — CHANGES_REQUESTED** [review]

**Review by boojack — APPROVED** [review]
LGTM

**Comment by CorrectRoadH** [comment]
@Zeng1998 PTAL

**Review by Zeng1998 — APPROVED** [review]
LGTM

**Pull request merged**